src/core/utils/EventManager.ts

```diff
@@ -1,7 +1,8 @@
 import { EventNormalizer } from './EventNormalizer';
+import { Signal, Action } from '../types/Flux';
 
 export interface EventManagerOptions {
-    dispatch?: Function;
+    dispatch?: (action: Action) => void;
```

Comment:
Maybe the Dispatcher should ```export type DispatchFunction = (action: Action) => void``` to make it explicit this cannot be any function but *has* to be the one actually dispatching ?